### ISSUE TITLE
Build failure with open-ce 1.3.0 + python 3.9 + cuda 11.2 + ppc64le

### ISSUE THREAD

**Issue by zhyizhyi** [open-ce/open-ce#471]
Using docker build, for open-ce 1.3.0 + python 3.9 + cuda 11.2 + ppc64le
Got build error
```
bazel --bazelrc=$SRC_DIR/tensorflow/tensorflow.bazelrc build \
    --local_cpu_resources=HOST_CPUS-10 \
    --local_ram_resources=HOST_RAM*0.50 \
    --jobs 32 \
    --config=opt \
    --config=numa \
    --curses=no \
    //tensorflow/tools/pip_package:build_pip_package
+ bazel --bazelrc=/opt/conda/conda-bld/tensorflow-base_1627677885231/work/tensorflow/tensorflow.bazelrc build --local_cpu_resources=HOST_CPUS-10 '--local_ram_resources=HOST_RAM*0.50' --jobs 32 --config=opt --config=numa --curses=no //tensorflow/tools/pip_package:build_pip_package
Starting local Bazel server and connecting to it...
ERROR: While parsing option --local_cpu_resources=HOST_CPUS-10: Value '(-6)' must be at least 1.
Traceback (most recent call last):
  File "/home/builder/open_ce/build_feedstock.py", line 150, in build_feedstock_from_command
    conda_build.api.build(os.path.join(os.getcwd(), recipe['path']),
  File "/opt/conda/lib/python3.9/site-packages/conda_build/api.py", line 186, in build
    return build_tree(
  File "/opt/conda/lib/python3.9/site-packages/conda_build/build.py", line 3068, in build_tree
    packages_from_this = build(metadata, stats,
  File "/opt/conda/lib/python3.9/site-packages/conda_build/build.py", line 2351, in build
    newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
  File "/opt/conda/lib/python3.9/site-packages/conda_build/build.py", line 1596, in bundle_conda
    utils.check_call_env(interpreter_and_args + [dest_file],
  File "/opt/conda/lib/python3.9/site-packages/conda_build/utils.py", line 412, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/conda_build/utils.py", line 392, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['/usr/bin/bash', '-e', '/opt/conda/conda-bld/tensorflow-base_1627677885231/work/build-pip-package.sh']' returned non-zero exit status 2.
[OPEN-CE-ERROR]-11 Unable to build recipe: tensorflow-feedstock
[OPEN-CE-ERROR]-11 Unable to build recipe: tensorflow-base
Command '['/usr/bin/bash', '-e', '/opt/conda/conda-bld/tensorflow-base_1627677885231/work/build-pip-package.sh']' returned non-zero exit status 2.
ERROR:OPEN-CE:[OPEN-CE-ERROR]-11 Unable to build recipe: tensorflow-feedstock
[OPEN-CE-ERROR]-11 Unable to build recipe: tensorflow-base
Command '['/usr/bin/bash', '-e', '/opt/conda/conda-bld/tensorflow-base_1627677885231/work/build-pip-package.sh']' returned non-zero exit status 2.
ERROR OPEN-CE:<module>(139): [OPEN-CE-ERROR]-11 Unable to build recipe: tensorflow-feedstock
[OPEN-CE-ERROR]-11 Unable to build recipe: tensorflow-base
Command '['/usr/bin/bash', '-e', '/opt/conda/conda-bld/tensorflow-base_1627677885231/work/build-pip-package.sh']' returned non-zero exit status 2.
open-ce-builder-20210730164355
```

**Comment by jayfurmanek** [comment]
This:
https://github.com/open-ce/tensorflow-feedstock/blob/main/recipe/build-pip-package.sh#L41-L42

should be smarter. We'll fix.

**Comment by bnemanich** [comment]
This was fixed by open-ce/tensorflow-feedstock#58
